go/client/cmd_encrypt.go

```diff
@@ -191,32 +209,62 @@ func (c *CmdEncrypt) GetUsage() libkb.Usage {
 func (c *CmdEncrypt) ParseArgv(ctx *cli.Context) error {
 	c.opts.TeamRecipients = ctx.StringSlice("team")
 	c.opts.Recipients = ctx.Args()
-	if len(c.opts.Recipients) == 0 && len(c.opts.TeamRecipients) == 0 {
-		return errors.New("Encrypt needs at least one recipient")
+	forRecipients := len(c.opts.Recipients) > 0
+	forTeamRecipients := len(c.opts.TeamRecipients) > 0
+	if forRecipients == forTeamRecipients { // !xor
```

Comment:
I would not nest here (and in general try to reduce nesting wherever possible). You can do 
```
if !forRecipients && !forTeamRecipients {
    return errors.New("need at least one recipient")
}
if forRecipients && forTeamRecipients {
    return errors.New("can only encrypt for either all individuals or all teams")
}
```

Reply:
Another thing you can do (only if you want) which would simplify things is the following: have a single `--team-recipients` boolean option: if it is passed then all the recipients are interpreted as teams, and if not they are interpreted as users. This way, when encrypting for many teams I only have to specify the `--team-recipients` once instead of `--team` many times